ml_service/pipelines/build_data_processing_os_cmd_pipeline.py

```diff
@@ -0,0 +1,94 @@
+from azureml.pipeline.core.graph import PipelineParameter
+from azureml.pipeline.steps import PythonScriptStep
+from azureml.pipeline.core import Pipeline
+from azureml.core import Workspace, Datastore
+from azureml.core.runconfig import RunConfiguration
+from azureml.data import OutputFileDatasetConfig
+from ml_service.util.attach_compute import get_compute
+from ml_service.util.env_variables import Env
+from ml_service.util.manage_environment import get_environment
+
+
+def main():
+    e = Env()
+    # Get Azure machine learning workspace
+    aml_workspace = Workspace.get(
+        name=e.workspace_name,
+        subscription_id=e.subscription_id,
+        resource_group=e.resource_group,
+    )
+    print(f"get_workspace:{aml_workspace}")
+
+    # Get Azure machine learning cluster
+    aml_compute = get_compute(aml_workspace, e.compute_name, e.vm_size)
+    if aml_compute is not None:
+        print(f"aml_compute:{aml_compute}")
+
+    # Create a reusable Azure ML environment
+    environment = get_environment(
+        aml_workspace,
+        e.aml_custom_docker_env_name,
+        create_new=e.rebuild_env,
+        enable_docker=True,
+        dockerfile=e.aml_custom_dockerfile
```

Comment:
Here we create the environment to use our custom docker. It is possible to install any tool to it.